src/test/java/seedu/address/logic/commands/UntagCommandTest.java

```diff
@@ -68,6 +68,18 @@ public void execute_invalidIndex_throwsCommandException() {
         assertCommandFailure(tagCommand, model, Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
     }
 
+    @Test
+    public void execute_tagNotExist_throwsCommandException() {
+
+        showPersonAtIndex(model, INDEX_FIRST_PERSON);
+
+        Tag tag = new Tag("friendss");
```

Comment:
possibly create a util for creating Tags for improving testability in the future?